spaghetto/bin/src/Main.purs

```diff
@@ -133,10 +157,13 @@ type BundleArgs =
   , pedanticPackages :: Boolean
   , type :: Maybe String
   , ensureRanges :: Boolean
+  , jsonErrors :: Boolean
+  , psaArgs :: PsaArgs
   }
 
 type PublishArgs =
   { selectedPackage :: Maybe String
+  , psaArgs :: PsaArgs
```

Comment:
Maybe the publishing build should not be configurable? I mean, the compiler is going to build a few times in different configurations, so not sure the user should look at that at all - we can probably skip these flags (the idea is that every flag adds cognitive weight to folks using a command, and this is important for `publish`) and give sensible defaults when we build the environment

Reply:
If `spago build` normally uses `--strict`, I'd expect `spago publish` to operate likewise (e.g. `--strict` will fail publishing if warnings exist). I think the stash file should be used as otherwise some compiler warnings that existed in a previous `build` may not be reported by `purs` a second time.